version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.5.0"
+	Version = "1.6.0"
```

Comment:
hm, we will create release branch before or after this commit?
part of changes needed for release branch, but this line is not needed for release branch

Reply:
The intention was to merge this and then to create release branch. I'll remove this, but what about change in `deploy/crd.yaml` - should it be removed also?